handler/handler.go

```diff
@@ -204,24 +204,16 @@ func (h *InstanceComplete) ParseQueryResponse(resp *cantabular.StaticDatasetQuer
 	b := new(bytes.Buffer)
 	w := csv.NewWriter(b)
 
-	// aux func to write to the csv writer, returning any error (returned by w.Write or w.Error)
-	write := func(record []string) error {
```

Comment:
Looks like you have some inadvertent merge issues here, I know this stuff was just added by David I doubt you meant to remove it?

Reply:
Hmm ...
Yes, i'll undo and put write() and its use back.